dev/bin/setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+ set -e
+
+ REPO_ROOT=$(git rev-parse --show-toplevel)
+
+ source ${REPO_ROOT}/dev/vendor/galoy-quickstart/bin/helpers.sh
+ source ${REPO_ROOT}/dev/.env
+
+ login_user "alice" "$ALICE_PHONE" "000000"
+ echo "alice logged in"
+ ALICE_TOKEN=$(read_value "alice")
+ receive_onchain
+ echo "alice funded"
+
+ login_user "alice" "$BOB_PHONE" "000000"
```

Comment:
should this be "bob"

Reply:
The next line, receive_onchain declared in quickstart adds money to token "alice"'s wallet. It doesn't take a parameter unfortunately. Once it gets fixed in upstream, it could be incorporated here.